sympy/solvers/tests/test_solveset.py

```diff
@@ -709,21 +709,9 @@ def test_solveset_complex_tan():
 
 def test_solve_trig():
     from sympy.abc import n
-    assert solveset_real(sin(x), x) == \
```

Comment:
I [ commented ](https://github.com/sympy/sympy/pull/10482/files#diff-85baa04bbf4e1dfd9128782738e45424R1059) it